src/scss/components/radio/_index.scss

```diff
@@ -154,88 +141,61 @@ hx-radio-control {
     }
 
     &:valid ~ label > hx-radio:hover {
-      background-color: $cyan-50;
-      border: 1px solid currentColor;
-      color: $cyan-500;
+      @include hxRadioFacade($pseudo-state: hover);
     }
   }
 }
 
 // INVALID STATE
 hx-radio-set[hx-dirty] input[type="radio"]:enabled:invalid,
 hx-radio-set.hxInvalid input[type="radio"]:enabled {
-  color: $red-900;
-
-  &::-ms-check {
-    border-width: 2px;
-  }
+  @include hxRadioControl(invalid);
 
   ~ label > hx-radio {
-    border: 2px solid currentColor;
-    color: $red-900;
+    @include hxRadioFacade(invalid);
   }
 
   &:focus {
-    box-shadow: $focus-glow-invalid;
+    @include hxRadioControl(invalid, focus);
 
     ~ label > hx-radio {
-      box-shadow: $focus-glow-invalid;
+      @include hxRadioFacade(invalid, focus);
     }
   }
 
   // native/ie11 styles
-  &:hover::-ms-check {
-    background-color: $red-100;
-    border: 2px solid currentColor;
-    color: $red-900;
```

Comment:
These styles need to be added to the `hxRadioControl` mixin. See comments for `src/scss/components/radio/mixins/_hxRadioControl.scss`.

Reply:
Changes have been done, please review.